app/src/components/NavigationTab/index.tsx

```diff
@@ -188,6 +188,7 @@ function NavigationTab(props: Props) {
             )}
             <div className={styles.childrenWrapper}>
                 {children}
+                <div className={styles.activeChildrenBorder} />
```

Comment:
1. Not sure if this is the right approach.
2. This should only be added on certain variants.

Reply:
It's only for primary variant. Styles condition are in corresponding styles file